azure-toolkit-libs/azure-toolkit-appservice-lib/src/main/resources/schema/appservice/AppServicePlan.json

```diff
@@ -0,0 +1,33 @@
+{
+  "$schema": "http://json-schema.org/draft-07/schema#",
+  "title": "App Service Plan - Update",
+  "description": "Schema for AppServicePlanConfig when update existing service plan",
+  "properties": {
+    "subscriptionId": {
+      "$ref": "classpath:///schema/common/UUID.json"
+    },
+    "servicePlanResourceGroup": {
+      "$ref": "classpath:///schema/common/ResourceGroupName.json"
+    },
+    "servicePlanName": {
+      "$ref": "classpath:///schema/appservice/AppServicePlanName.json"
+    },
+    "region": {
+      "$ref": "classpath:///schema/common/NonEmptyString.json"
+    },
+    "os": {
+      "$ref": "classpath:///schema/appservice/Runtime.json#/definitions/os"
+    },
+    "region": {
+      "type": "object"
```

Comment:
why two region?

Reply:
type object?

Reply:
Region contains two properties, value and display name, so it will convert to json object when serialize. And the duplicated property has been removed